.github/workflows/wheels-build.yaml

```
@@ -20,19 +20,23 @@ on:
         required: true
         type: string
       package-type:
+        description: "One of: [cpp, python]"
         required: false
         type: string
       wheel-name:
         required: false
         type: string
+        description: "The name of the wheel"
       pure-wheel:
         required: false
         type: boolean
         default: false
+        description: "One of [true, false] to indicate if the wheel is a pure wheel"
```

```suggestion
        description: "One of [true, false], true if the wheel is not dependent on operating system, Python version, or CPU architecture"
```

For someone who was already unsure about the name `pure-wheel`, I think that saying "is a pure wheel" in the description wouldn't help them to make a choice here. Would you consider this alternative wording?